main.go

```diff
@@ -211,15 +224,19 @@ func NewApp() *cli.App {
 	return app
 }
 
-// load environment variables from .env file
-func loadDotEnv() {
-	if _, err := os.Stat(".env"); err != nil {
-		return
+// load environment variables
+func loadDotEnv(filenames ...string) error {
+	for _, filename := range filenames {
+		if _, err := os.Stat(filename); err != nil {
+			return nil
+		}
 	}
 
-	if err := godotenv.Load(); err != nil {
-		log.Fatalf("Error loading .env file: %s", err.Error())
+	if err := godotenv.Load(filenames...); err != nil {
+		return fmt.Errorf("Error loading env file: %s", err.Error())
 	}
+
+	return nil
 }
```

Comment:
We can check for the error condition where it is `os.ErrNotExist` and ignore it here, otherwise return the error which our caller can handle as appropriate:

```
// load environment variables from files
func loadDotEnv(filenames ...string) error {
	err := godotenv.Load(filenames...)
	if err != nil {
		var perr *os.PathError
		if errors.As(err, &perr) {
			if errors.Is(perr, os.ErrNotExist) {
				// this is fine, ignore this error
				return nil
			}
		}
		return fmt.Errorf("loading env file(s) %v: %v", filenames, err)
	}

	return nil
}
```